src/Backoffice/Security/Permissions/Application/Search/PermissionsSearcher.php

```
@@ -0,0 +1,70 @@
+                ->where('name', $permissionName)
+                ->first();
+
+            if ($permission) {
+                // Check if the role has this permission
+                $hasPermission = DB::table('role_has_permissions')
+                    ->where('permission_id', $permission->id)
+                    ->where('role_id', $roleId)
+                    ->exists();
+
+                $permissions[] = [
+                    'id' => $permission->id,
```

PHPStan (2.0.3)

Access to an undefined property object::$id.
Learn more: https://phpstan.org/blog/solving-phpstan-access-to-undefined-property

(property.notFound)